### PR TITLE
Adds Medpouches (and other Bay Stuff)

### PR DESCRIPTION
**Pull request by alsoandanswer**
For WickedCybs

Adds medpouches and adjusts first-aid kits to their Bay equivalents.

Q&A
- The Autoinjectors/pills are missing!
Each medpouch comes with a pill and autoinjector. Technically this is a buff across the board.

Also : Renames cough syrup's chemical name to antidexafen. To match Bay counterpart. Most references are still cough syrup.
Adds radiation first aid kits to radiation closets. Not like anyone has trouble with radiation, but...

_(edited by SleepyGemmy @ 2023-02-07: removes mention of health analyzer removal.)_

### CODE REVIEW

**Comment by DreamySkrell** [comment]
> The health analysers are removed!
They'll still be in the Nanomed Plus (a.k.a, the Big One), which most of medical uses anyway.
This mostly removes them from first-aid kits and the small wall mounted ones.
(which face it, people rarely use, and the ones who need it, medical, already have it. Unless you're a powergamer.)

This is going to hurt third party ships, antags, and the like. Especially third party ships who do not have full medbays, and might need to rely on things like random crate supplies (first aid kits are pretty common there in my experience), or even just mapped in first aid kits.

> (which face it, people rarely use, and the ones who need it, medical, already have it. Unless you're a powergamer.)

Carrying and using a health analyzer is not powergaming. It's basic equipment that anyone can use. We even have an internal health analyzer augment that anyone can pick.

**Comment by SleepyGemmy** [comment]
i second what dreamy wrote.

**Comment by TheDanseMacabre** [comment]
I don't see any reason to make health analyzers less common/harder to attain. They should definitely still be in the wall-mounted nanomeds and I still think they should spawn in FAKs.

**Comment by alsoandanswer** [comment]
!review

**Comment by alsoandanswer** [comment]
I guess I'll have to elaborate.
**On Health Analyzers : It shoudn't be basic equipment.**
I believe the current examine is sufficient; you already can check pulse, bleeding, brute, burn without health analyzers.
Which is sufficient to *stabilize* anyone. You shouldn't be _anal-retentive_ about getting the guy to 100% health.
You shouldn't be able to save someone from the brink of death.
_That's medical's job. In my opinion, it is absolutely powergaming._ 

As for blood oxygenation, brain activity, toxins etc., if you aren't medical, 
most likely your character will not know this information, nor how to use it in any way.

On augments : This is an entirely different can of worms on balance and as such should not be used as a baseline.
In my honest opinion, really only medical should have access to health analysers.
(even then, they don't need it since it comes pre-loaded in their PDAs.)

**Comment by SleepyGemmy** [comment]
i would recommend making a seperate atomic PR removing health analyzers.

**Review by NonQueueingMatt — CHANGES_REQUESTED** [review]
Health analyzers should stay in normal medkits for now. Split that removal into a different PR.

When you do make the PR to remove them, make sure to keep them in first aid kits used for third party ships.

**Comment by alsoandanswer** [comment]
!review

**Review by SleepyGemmy — DISMISSED** [review]
previous [review](https://github.com/Aurorastation/Aurora.3/pull/15505#pullrequestreview-1247480455) not addressed.

**Review by NonQueueingMatt — CHANGES_REQUESTED** [review]
Health analyzers will stay in medkits in this PR. Re-add them.

Reason being that health analyzers are not actually specialised equipment - see the health analyzer augment that anyone can take.

**Review by SleepyGemmy — CHANGES_REQUESTED** [review]
ditto my last review.

**Review by NonQueueingMatt — APPROVED** [review]

File: code/game/machinery/vending_types.dm [hunk 1]
```diff
@@ -591,10 +591,13 @@
...
-		/obj/item/stack/medical/ointment = 2,
-		/obj/item/reagent_containers/hypospray/autoinjector/inaprovaline = 4,
-		/obj/item/reagent_containers/hypospray/autoinjector/dylovene = 4,
+		/obj/item/stack/medical/bruise_pack = 3,
+		/obj/item/stack/medical/ointment = 3,
+		/obj/item/reagent_containers/pill/perconol = 4,
+		/obj/item/storage/box/fancy/med_pouch/trauma,
+		/obj/item/storage/box/fancy/med_pouch/burn,
+		/obj/item/storage/box/fancy/med_pouch/oxyloss,
+		/obj/item/storage/box/fancy/med_pouch/toxin,
```
Comment (SleepyGemmy) [inline code/game/machinery/vending_types.dm:600]:
```suggestion
		/obj/item/storage/box/fancy/med_pouch/trauma = 1,
		/obj/item/storage/box/fancy/med_pouch/burn = 1,
		/obj/item/storage/box/fancy/med_pouch/oxyloss = 1,
		/obj/item/storage/box/fancy/med_pouch/toxin = 1,
```

File: code/game/machinery/vending_types.dm [hunk 2]
```diff
@@ -620,10 +623,15 @@
...
-		/obj/item/stack/medical/bruise_pack = 3,
-		/obj/item/stack/medical/ointment = 3,
-		/obj/item/device/healthanalyzer = 3
+		/obj/item/stack/medical/bruise_pack = 4,
+		/obj/item/stack/medical/ointment = 4,
+		/obj/item/storage/box/fancy/med_pouch/trauma,
+		/obj/item/storage/box/fancy/med_pouch/burn,
+		/obj/item/storage/box/fancy/med_pouch/oxyloss,
+		/obj/item/storage/box/fancy/med_pouch/toxin,
+		/obj/item/storage/box/fancy/med_pouch/radiation,
```
Comment (SleepyGemmy) [inline code/game/machinery/vending_types.dm:632]:
```suggestion
		/obj/item/storage/box/fancy/med_pouch/trauma = 1,
		/obj/item/storage/box/fancy/med_pouch/burn = 1,
		/obj/item/storage/box/fancy/med_pouch/oxyloss = 1,
		/obj/item/storage/box/fancy/med_pouch/toxin = 1,
		/obj/item/storage/box/fancy/med_pouch/radiation = 1,
```

File: code/game/machinery/vending_types.dm [hunk 3]
```diff
@@ -620,10 +623,15 @@
...
+		/obj/item/device/healthanalyzer = 1,
+		/obj/item/device/breath_analyzer  = 1
```
Comment (SleepyGemmy) [inline code/game/machinery/vending_types.dm:634]:
```suggestion
		/obj/item/device/breath_analyzer = 1
```

File: code/game/objects/items/weapons/storage/firstaid.dm [hunk 4]
```diff
@@ -17,50 +8,44 @@
...
+	desc = "It's an emergency medical kit for people who like wish soup."
+
+/obj/item/storage/firstaid/regular
+	starts_with = list(
+		/obj/item/stack/medical/bruise_pack = 2,
+		/obj/item/stack/medical/ointment = 2,
+		/obj/item/storage/pill_bottle/antidexafen = 1,
+		/obj/item/storage/pill_bottle/perconol = 1,
+		/obj/item/stack/medical/splint = 1,
+		/obj/item/device/healthanalyzer
```
Comment (SleepyGemmy) [inline code/game/objects/items/weapons/storage/firstaid.dm:26]:
```suggestion
		/obj/item/device/healthanalyzer = 1
```

File: code/game/objects/items/weapons/storage/firstaid.dm [hunk 5]
```diff
@@ -88,10 +71,11 @@
 	icon_state = "advfirstaid"
 	item_state = "advfirstaid"
 	starts_with = list(
-		/obj/item/reagent_containers/hypospray/autoinjector/inaprovaline = 1,
+		/obj/item/storage/pill_bottle/assorted = 1,
 		/obj/item/stack/medical/advanced/bruise_pack = 3,
 		/obj/item/stack/medical/advanced/ointment = 2,
-		/obj/item/stack/medical/splint = 1
+		/obj/item/stack/medical/splint = 1,
+		/obj/item/device/healthanalyzer
```
Comment (SleepyGemmy) [inline code/game/objects/items/weapons/storage/firstaid.dm:78]:
```suggestion
		/obj/item/device/healthanalyzer = 1
```

File: code/game/objects/items/weapons/storage/firstaid.dm [hunk 6]
```diff
@@ -110,7 +94,8 @@
 		/obj/item/storage/pill_bottle/dylovene = 1,
 		/obj/item/storage/pill_bottle/mortaphenyl = 1,
 		/obj/item/reagent_containers/hypospray/autoinjector/sideeffectbgone = 1,
-		/obj/item/stack/medical/splint = 1
+		/obj/item/stack/medical/splint = 1,
+		/obj/item/device/healthanalyzer
```
Comment (SleepyGemmy) [inline code/game/objects/items/weapons/storage/firstaid.dm:98]:
```suggestion
		/obj/item/device/healthanalyzer = 1
```

File: code/game/objects/items/weapons/storage/firstaid.dm [hunk 7]
```diff
@@ -133,184 +118,84 @@
...
+	name = "stabilisation first-aid"
+	desc = "Stocked with medical pouches."
+	icon_state = "firstaid_multi"
+	starts_with = list(
+		/obj/item/storage/box/fancy/med_pouch/trauma = 1,
+		/obj/item/storage/box/fancy/med_pouch/burn = 1,
+		/obj/item/storage/box/fancy/med_pouch/oxyloss = 1,
+		/obj/item/storage/box/fancy/med_pouch/toxin = 1,
+		/obj/item/storage/box/fancy/med_pouch/radiation = 1,
+		/obj/item/device/healthanalyzer
```
Comment (SleepyGemmy) [inline code/game/objects/items/weapons/storage/firstaid.dm:161]:
```suggestion
		/obj/item/device/healthanalyzer = 1
```

File: code/game/objects/items/weapons/storage/firstaid.dm [hunk 8]
```diff
@@ -133,184 +118,84 @@
...
+	w_class = ITEMSIZE_SMALL
+	storage_slots = 7
+	starts_with = list(
+		/obj/item/reagent_containers/hypospray/autoinjector/coagzolug = 1,
+		/obj/item/reagent_containers/hypospray/autoinjector/pain = 1,
+		/obj/item/reagent_containers/hypospray/autoinjector/pouch_auto/adrenaline = 1,
+		/obj/item/reagent_containers/hypospray/autoinjector/pouch_auto/inaprovaline = 2,
+		/obj/item/reagent_containers/hypospray/autoinjector/oxygen = 1,
+		/obj/item/reagent_containers/hypospray/autoinjector/dylovene = 1,
+		/obj/item/device/healthanalyzer
```
Comment (SleepyGemmy) [inline code/game/objects/items/weapons/storage/firstaid.dm:178]:
```suggestion
		/obj/item/device/healthanalyzer = 1
```

File: code/game/objects/items/weapons/storage/med_pouch.dm [hunk 9]
```diff
@@ -0,0 +1,185 @@
...
+/obj/item/storage/box/fancy/med_pouch
+	name = "emergency medical pouch"
+	desc = "For use in emergency situations only."
+	desc_info = "\
+	1) Tear open the emergency medical pack using the easy open tab at the top.<br>\
+	2) Carefully remove all items from the pouch and discard the pouch.<br>\
+	3) Apply all autoinjectors to the injured party.<br>\
+	4) Use bandages to stop bleeding if required.<br>\
+	5) Force the injured party to swallow all pills.<br>\
+	6) Use ointment on any burns if required<br>\
```
Comment (SleepyGemmy) [inline code/game/objects/items/weapons/storage/med_pouch.dm:14]:
```suggestion
	6) Use ointment on any burns if required.<br>\
```

File: code/game/objects/items/weapons/storage/med_pouch.dm [hunk 10]
```diff
@@ -0,0 +1,185 @@
...
+	6) Contact the medical team with your location.<br>\
+	7) Stay in place once they respond."
+	injury_type = "trauma"
+	color = COLOR_RED
+
+	starts_with = list(
+	/obj/item/reagent_containers/hypospray/autoinjector/pouch_auto/inaprovaline = 1,
+	/obj/item/reagent_containers/pill/pouch_pill/inaprovaline = 1,
+	/obj/item/reagent_containers/pill/pouch_pill/perconol = 1,
+	/obj/item/stack/medical/bruise_pack = 2
```
Comment (SleepyGemmy) [inline code/game/objects/items/weapons/storage/med_pouch.dm:56]:
```suggestion
		/obj/item/reagent_containers/hypospray/autoinjector/pouch_auto/inaprovaline = 1,
		/obj/item/reagent_containers/pill/pouch_pill/inaprovaline = 1,
		/obj/item/reagent_containers/pill/pouch_pill/perconol = 1,
		/obj/item/stack/medical/bruise_pack = 2
```

File: code/game/objects/items/weapons/storage/med_pouch.dm [hunk 11]
```diff
@@ -0,0 +1,185 @@
...
+	name = "burn pouch"
+	injury_type = "burn"
+	color = COLOR_SEDONA
+
+	starts_with = list(
+	/obj/item/reagent_containers/hypospray/autoinjector/pouch_auto/inaprovaline = 1,
+	/obj/item/reagent_containers/hypospray/autoinjector/pouch_auto/mortaphenyl = 1,
+	/obj/item/reagent_containers/hypospray/autoinjector/pouch_auto/adrenaline = 1,
+	/obj/item/reagent_containers/pill/pouch_pill/perconol = 1,
+	/obj/item/stack/medical/ointment = 2
```
Comment (SleepyGemmy) [inline code/game/objects/items/weapons/storage/med_pouch.dm:69]:
```suggestion
		/obj/item/reagent_containers/hypospray/autoinjector/pouch_auto/inaprovaline = 1,
		/obj/item/reagent_containers/hypospray/autoinjector/pouch_auto/mortaphenyl = 1,
		/obj/item/reagent_containers/hypospray/autoinjector/pouch_auto/adrenaline = 1,
		/obj/item/reagent_containers/pill/pouch_pill/perconol = 1,
		/obj/item/stack/medical/ointment = 2
```

File: code/game/objects/items/weapons/storage/med_pouch.dm [hunk 12]
```diff
@@ -0,0 +1,185 @@
...
+	name = "low oxygen pouch"
+	injury_type = "low oxygen"
+	color = COLOR_BLUE
+
+	starts_with = list(
+	/obj/item/reagent_containers/hypospray/autoinjector/pouch_auto/inaprovaline = 1,
+	/obj/item/reagent_containers/hypospray/autoinjector/pouch_auto/dexalin = 1,
+	/obj/item/reagent_containers/hypospray/autoinjector/pouch_auto/adrenaline = 1,
+	/obj/item/reagent_containers/pill/pouch_pill/inaprovaline = 1,
+	/obj/item/reagent_containers/pill/pouch_pill/dexalin = 1
```
Comment (SleepyGemmy) [inline code/game/objects/items/weapons/storage/med_pouch.dm:91]:
```suggestion
		/obj/item/reagent_containers/hypospray/autoinjector/pouch_auto/inaprovaline = 1,
		/obj/item/reagent_containers/hypospray/autoinjector/pouch_auto/dexalin = 1,
		/obj/item/reagent_containers/hypospray/autoinjector/pouch_auto/adrenaline = 1,
		/obj/item/reagent_containers/pill/pouch_pill/inaprovaline = 1,
		/obj/item/reagent_containers/pill/pouch_pill/dexalin = 1
```

File: code/game/objects/items/weapons/storage/med_pouch.dm [hunk 13]
```diff
@@ -0,0 +1,185 @@
...
+	8) Stay in place once they respond."
+
+/obj/item/storage/box/fancy/med_pouch/toxin
+	name = "toxin pouch"
+	injury_type = "toxin"
+	color = COLOR_GREEN
+
+	starts_with = list(
+	/obj/item/reagent_containers/hypospray/autoinjector/pouch_auto/dylovene = 1,
+	/obj/item/reagent_containers/pill/pouch_pill/dylovene = 1
```
Comment (SleepyGemmy) [inline code/game/objects/items/weapons/storage/med_pouch.dm:110]:
```suggestion
		/obj/item/reagent_containers/hypospray/autoinjector/pouch_auto/dylovene = 1,
		/obj/item/reagent_containers/pill/pouch_pill/dylovene = 1
```

File: code/game/objects/items/weapons/storage/med_pouch.dm [hunk 14]
```diff
@@ -0,0 +1,185 @@
...
+	6) Stay in place once they respond."
+
+/obj/item/storage/box/fancy/med_pouch/radiation
+	name = "radiation pouch"
+	injury_type = "radiation"
+	color = COLOR_WHEAT
+
+	starts_with = list(
+	/obj/item/reagent_containers/hypospray/autoinjector/hyronalin = 1,
+	/obj/item/reagent_containers/pill/pouch_pill/dylovene = 1
```
Comment (SleepyGemmy) [inline code/game/objects/items/weapons/storage/med_pouch.dm:127]:
```suggestion
		/obj/item/reagent_containers/hypospray/autoinjector/hyronalin = 1,
		/obj/item/reagent_containers/pill/pouch_pill/dylovene = 1
```

**Review by SleepyGemmy — CHANGES_REQUESTED** [review]

**Review by SleepyGemmy — APPROVED** [review]

**Pull request merged**